app/config/mq-config.js

```diff
@@ -18,6 +18,10 @@ const mqSchema = joi.object({
   processingTopic: {
     name: joi.string(),
     address: joi.string()
+  },
+  eventTopic: {
+    name: joi.string(),
```

Comment:
Don't think we need name on this as we don't use it for anything.

Reply:
removed